src/RolloutScorer/RolloutScorer.Service/RolloutScorerProcessor.cs

```diff
@@ -1,37 +1,45 @@
 using Microsoft.Azure.KeyVault;
```

Comment:
Whole file was moved from the Azure Function and will need to be refactored to be a proper SF service.